app/Phragile/Burndown.php

```diff
@@ -0,0 +1,107 @@
+<?php
+namespace Phragile;
+
+class Burndown {
+	// TODO: ideally these should come from a cached call to maniphest.querystatuses
+	private static $STATUS_OPEN = ['stalled', 'open'];
+
+	public function __construct(\Sprint $sprint, TaskList $tasks, PhabricatorAPI $phabricator)
+	{
+		$this->sprint = $sprint;
+		$this->tasks = $tasks;
+		$this->phabricator = $phabricator;
+		$this->days = $this->daysInSprint();
+	}
+
+	private function daysInSprint()
+	{
+		$days = [];
+
+		for ($day = strtotime($this->sprint->sprint_start);
```

Comment:
Is there a reason you have public properties (as opposed to getters, and perhaps setters where applicable) in Sprint?